config/authors/karunganessh.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Arun Ganessh",
+  "bio": "",
```

Comment:
Arun has 4+ years of experience in Developing Android, AR/MR/VR Applications using java, C, C++ language. In addition, he has 1+ years of experience in SAS, Oracle Apex, SQL, HTML, Opengl c++ ,Vulkan. Currently a Lead Engineer at Boeing.

Reply:
Updated